### PR TITLE
feat: add v custom build

### PR DESCRIPTION
**Pull request by medgar-nitric**

### CODE REVIEW

**Review by tjholm — COMMENTED** [review]
I think Jye had some interesting configuration comparisons for this, like cloudformation vs. nitric.yaml to really showcase the difference in volume of configuration and knowledge required as well.

**Comment by medgar-nitric** [comment]
> I think Jye had some interesting configuration comparisons for this, like cloudformation vs. nitric.yaml to really showcase the difference in volume of configuration and knowledge required as well.

Sounds good if you want to add that content, that would be great.

File: docs/intro/vs/custom.mdx [hunk 1]
```diff
@@ -1 +1,24 @@
...
+When evaluating Nitric against a fully in-house custom build, it is important to consider the total cost of ownership of these options. Also when not choosing Nitric there are likely to be serious down stream implications for the business which need to be understood.
+
+Nitric provides a comprehensive set of features for the rapid development of multi-cloud solutions including:
+
+- automated multi-cloud deployment and CLI
+- declarative stack definitions
+- Membrane multi-cloud application gateway/adaptor
+- multi-cloud service APIs
+- FaaS micro-framework
+- supports multi development languages
```
Comment (HomelessDinosaur) [inline docs/intro/vs/custom.mdx:10]: multi -> multiple

File: docs/intro/vs/custom.mdx [hunk 2]
```diff
@@ -1 +1,24 @@
...
+- comprehensive documentation
+
+For most organizations developing these capabilities in house is a non-starter because:
+
+- prohibitive time and cost of this development
+- difficulty of attracting the require development staff
+- ongoing maintenance and support costs
+- focus on undifferentiated heavy lifting rather than the Business objectives
+
+Making the choice for Custom Build over Nitric will prematurely reduces Optionality for the business, this may include:
```
Comment (HomelessDinosaur) [inline docs/intro/vs/custom.mdx:21]: reduces -> reduce

File: docs/intro/vs/custom.mdx [hunk 3]
```diff
@@ -1 +1,24 @@
...
+
+- only supporting one Cloud vendor, limiting future business sales options and reduces cloud pricing control
```
Comment (HomelessDinosaur) [inline docs/intro/vs/custom.mdx:23]: , limiting -> limits ?

File: docs/intro/vs/custom.mdx [hunk 4]
```diff
@@ -1 +1,24 @@
...
+- multi-cloud service APIs
+- FaaS micro-framework
+- supports multiple development languages
+- project templates
+- comprehensive documentation
+
+For most organizations developing these capabilities in house is a non-starter because:
+
+- prohibitive time and cost of this development
+- difficulty of attracting the require development staff
```
Comment (davemooreuws) [inline docs/intro/vs/custom.mdx:17]: require -> required

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by HomelessDinosaur — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**